projects/explorer/src/app/pages/blocks/blocks.component.ts

```diff
@@ -20,27 +20,40 @@ export class BlocksComponent implements OnInit {
   defaultPageSize = this.pageSizeOptions[1];
   defaultPageNumber = 1;
 
-  pollingInterval = 30;
+  pollingInterval = 5;
   latestBlock$: Observable<InlineResponse20035 | undefined>;
   latestBlockHeight$: Observable<bigint | undefined>;
   firstBlockHeight$: Observable<bigint | undefined>;
   blocks$: Observable<InlineResponse20036[] | undefined>;
 
+  autoEnabled: BehaviorSubject<boolean> = new BehaviorSubject<boolean>(false);
+  blocksEmpty: boolean;
```

Comment:
When you name a variable returns a boolean value, it would be a good idea to create a description like "If auto update is enabled,". In this case, I think `isEnabledAutoUpdate` is better.

Reply:
This is not fixed, But it's OK.